lib/screens/explore_screens/vote_screens/vote/interactor/viewmodels/vote_state.dart

```diff
@@ -3,16 +3,18 @@ part of 'vote_bloc.dart';
 class VoteState extends Equatable {
   final PageState pageState;
   final String? errorMessage;
-  final int remainingTimeStamp;
+  final int cycleEndTimestamp;
```

Comment:
need to make really clear that this does not update. Hence the name change

The next PR will make it even more clear. 

We used to update this with a time remaining seconds value, but it's much more stable and better to simply store the end time stamp here. 

The code has been storing the end timestamp for a while now, but the variable name was still the old variable name.